### DISCUSSION TITLE
Results for the #_ATTENDEES placeholder

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#405] [Q&A] [answered]
Hi Franky,
When I use #_ATTENDEES. I get the list of people who have booked, including the waiting list. I would like to have only the approved reservations or the specific mention that this person is on the waiting list.
How do I get this?
and by the way, happy new year to you !

**Comment by liedekef** [comment]
Don't use ATTENDEES then, but use [eme_attendees rsvp_status=1], see https://www.e-dynamics.be/wordpress/656/

**Comment by f14m07** [comment]
Well I would like to use this in 'Default event list format' and 'Default single event format' I can't use then the short code

**Comment by liedekef** [comment]
And why would that be? You can use shortcodes in there.

**Comment by f14m07** [comment]
When I define the format in Setting/Event/Default single event format, If I use the shortcode there The display is disturbed

**Reply by liedekef** [reply]
And which display is not ok? Because when viewing a single event, both #_ATTENDEES and [eme_attendees] generate by default the same html (just tried it here again to make sure as well)

**Reply by f14m07** [reply]
This is my 'Default single event format' 
```
<strong> DESCRIPTION DE LA SORTIE</strong></br>
<div style="background: LightYellow">#_NOTES</div><br />
<strong> LES POINTS ESSENTIELS</strong></br>
<div style="background: LightCyan">Départ le <strong>#_STARTDATE à #_STARTTIME </strong>et retour le #_ENDDATE vers #_ENDTIME<br /> Vous naviguerez sur  <strong>#_LOCATIONNAME</strong> avec le chef de bord <strong>#_AUTHORDISPNAME </strong><br />
Participation demandée : <strong>#_EVENTPRICE</strong> <br />
#_TOTALSEATS places "équipier" sont offertes plus #_WAITINGLISTSEATS en liste d'attente <br />
pour l'instant #_APPROVEDSEATS réservations sont approuvées, et #_PENDINGSEATS est (sont) en liste d'attente [eme_attendees rsvp_status=1]<br /> il reste <SPAN style="color:red">#_AVAILABLESEATS place(s) disponible(s)</SPAN><br /><br />
</div></br>
````
and the result is in this picture, with the mention 'no responses yet'
<img width="799" alt="Capture d’écran 2024-01-05 à 13 47 23" src="https://github.com/liedekef/events-made-easy/assets/113473805/dbfec91e-df51-43b8-9fad-e4ad7a46e704">

and in the 'Default event list format'
```
<table style="width:100% padding-inline-start: 2px">
<tr>
    <td style="width:45%;border-radius: 5px;background-color:LightGreen">D <span style="color: blue">#_STARTDATE #_STARTTIME <br>R #_ENDDATE #_ENDTIME</span> <br> avec  <b>#_LOCATIONNAME</b><br>#_LOCATIONIMAGETHUMB <br></td>
    
<td style="width:55%;border-radius: 10px;background-color:LemonChiffon;padding: 2px"> Nom #_LINKEDNAME <br> chef de bord <b>#_AUTHORDISPNAME</b></br>
<i>#_NOTES</i><br>
il reste <SPAN style="color:red">#_AVAILABLESEATS </SPAN> place(s) disponible(s)</br>
#_APPROVEDSEATS places sont déjà réservée(s) par [eme_attendees rsvp_status=1] 
 </td>
 </tr>
</table>
````
The result is 
<img width="390" alt="Capture d’écran 2024-01-05 à 13 52 52" src="https://github.com/liedekef/events-made-easy/assets/113473805/35fca0f8-bfbb-4d56-9722-40398f1ad5ae">

maybe also it is a recurrencie

**Reply by liedekef** [reply, marked as answer]
Your last example is not for the single event format, but the event listing. The listing there is not ok since eme_attendees needs an event id in that case. In the event listing, change the call to:
[eme_attendees id=#_EVENTID rsvp_status=1]

**Reply by f14m07** [reply]
It's perfect now, thank you very much for your information.

**Comment by f14m07** [comment]
Hi,
Is it possible **not** to display 'No responses yet' when there is no bookings or no pending

**Comment by liedekef** [comment]
Either do it with eme_if or use css to hide the class "eme_no_bookings"

**Reply by f14m07** [reply]
I use this code 
```
[eme_if tag='#ESC_APPROVEDSEATS' gt='0']
#_APPROVEDSEATS places sont déjà réservée(s) par [eme_attendees id=#_EVENTID rsvp_status=1] 
[/eme_if]
[eme_if tag='#ESC_PENDINGSEATS' gt='0']et #_PENDINGSEATS est (sont) en liste d'attente [eme_attendees id=#_EVENTID rsvp_status=2] 
[eme_if]
```
and it works for approvedseats but not for pendingseats

**Reply by f14m07** [reply]
hiding the class "eme_no_bookings" with CSS is enough but the first solution would be better if it works with #_PENDINGSEATS

**Reply by liedekef** [reply]
What is not working with #_PENDINGSEATS?

**Reply by f14m07** [reply]
[eme_if tag='#ESC_PENDINGSEATS' gt='0'] does not seem to works and 'No responses yet'is still displayed

**Comment by liedekef** [comment]
The #_PENDINGSEATS call seems to be working fine here, I can't reproduce anything wrong with it.
The "No responses yet" is only shown if you have a call to show the list of attendees outside the eme_if part. Again no issue here.

**Reply by f14m07** [reply]
Ok a typo error from me.
Thanks a lot for your support
